Description: define c_args for valac compilation
 FTBFS due to global c_args being seemingly
 ignored.
 Not reported upstream so this appears to be a
 distro specific issue
Author: David Mohammed <fossfreedom@ubuntu.com>
Forwarded: not-needed
Last-Update: 2024-08-17

--- a/src/wm/meson.build
+++ b/src/wm/meson.build
@@ -86,6 +86,8 @@
     vala_args: budgie_wm_vala_args,
     c_args: [
         '-DGNOME_DESKTOP_USE_UNSTABLE_API',
+        '-Wno-implicit-function-declaration',
+        '-w'
     ],
     install_rpath: rpath,
 )
--- a/src/daemon/meson.build
+++ b/src/daemon/meson.build
@@ -93,6 +93,7 @@
     ],
     c_args: [
         '-DWNCK_I_KNOW_THIS_IS_UNSTABLE',
+        '-Wno-implicit-function-declaration'
     ],
     install: true,
 )
--- a/src/panel/applets/workspaces/meson.build
+++ b/src/panel/applets/workspaces/meson.build
@@ -51,6 +51,7 @@
     ],
     c_args: [
         '-DWNCK_I_KNOW_THIS_IS_UNSTABLE',
+        '-Wno-implicit-function-declaration'
     ],
     install: true,
     install_dir: applet_workspaces_dir,
